dev-haskell/svgcairo/svgcairo-0.13.2.1-r1.ebuild

```diff
@@ -1,41 +1,38 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-# ebuild generated by hackport 0.6.3
+# ebuild generated by hackport 0.9.0.0.9999
+
+CABAL_HACKAGE_REVISION=1
 
 CABAL_FEATURES="lib haddock hoogle hscolour" # drop USE=profile as __attribute__ mangling is not compatible with glibc
 inherit haskell-cabal
 
 DESCRIPTION="Binding to the libsvg-cairo library"
-HOMEPAGE="http://projects.haskell.org/gtk2hs/"
-SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+HOMEPAGE="https://projects.haskell.org/gtk2hs/"
 
 LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64"
-IUSE=""
 
 GHC_BOOTSTRAP_PACKAGES=(
 	gtk2hs-buildtools
 )
 
+PATCHES=(
+	"${FILESDIR}"/${P}-librsvg.patch
+	)
+
 RDEPEND=">=dev-haskell/cairo-0.13.0.0:= <dev-haskell/cairo-0.14:=
 	>=dev-haskell/glib-0.13.0.0:= <dev-haskell/glib-0.14:=
-	dev-haskell/mtl:=
 	dev-haskell/text:=
-	>=dev-lang/ghc-7.4.1:=
+	>=dev-lang/ghc-9.0.2:=
 	gnome-base/librsvg
 "
 DEPEND="${RDEPEND}
-	>=dev-haskell/cabal-1.24
+	>=dev-haskell/cabal-3.4.1.0 <dev-haskell/cabal-3.11
 	>=dev-haskell/gtk2hs-buildtools-0.13.2.0 <dev-haskell/gtk2hs-buildtools-0.14
 	virtual/pkgconfig
```

Comment:
These should go in BDEPEND. Hackport doesn't yet automate it, not the end of the world, but technically incorrect.